packages/salesforcedx-vscode-core/src/commands/forceSourceRetrieveSourcePath.ts

```diff
@@ -84,7 +90,42 @@ export class SourcePathChecker implements PostconditionChecker<string> {
   }
 }
 
-export async function forceSourceRetrieveSourcePath(explorerPath: vscode.Uri) {
+export const forceSourceRetrieveSourcePath = async (
+  explorerPath: vscode.Uri,
+  uris: vscode.Uri[]
+) => {
+  // In the case that only one file is retrieved,
+  // uris contains a single path - the same as explorerPath
+  // and isn't undefined.
+  if (uris && uris.length > 1) {
+    await forceSourceRetrieveMultipleSourcePaths(uris);
+  } else {
+    await forceSourceRetrieveSingleSourcePath(explorerPath);
+  }
+};
+
+export const forceSourceRetrieveMultipleSourcePaths = async (uris: vscode.Uri[]) => {
+  const messages: ConflictDetectionMessages = {
+    warningMessageKey: 'conflict_detect_conflicts_during_retrieve',
+    commandHint: input => {
+      return new SfdxCommandBuilder()
```

Comment:
I've seen this in other places as well. Since we are using LibraryRetrieveSourcePathExecutor, is this hint correct? Should add what SDR would expect or simply leave this empty?

Reply:
Thanks for bringing this up.  I've been experimenting with this and here's what I found:
* If I leave empty (and use `commandHint: input => ""`, in the terminal, the user sees this:
 ```


Conflicts:
    Found 1 file(s) in conflict:

BookTests.cls

Run the following command to overwrite the conflicts:
```

The message reported to the user says, "Run the following" but yet nothing follows, so I think we should give them the commands to manually run if they wish.
* We're using LibraryRetrieveSourcePathExecutor, but the user doesn't have direct access to this, and would need to use sfdx in their terminal, so my preference would be to give them the commands.
* While looking into this, I found a build (which I introduced) pertaining to multiple files.  I pushed the update to the forceSourceDeployMultipleSourcePaths, but not forceSourceRetrieveMultipleSourcePaths just yet (so we don't loose this conversation - once we decide on want to do, I'll get the approach in both forceSourceDeployMultipleSourcePaths and forceSourceRetrieveMultipleSourcePaths to match.

Reply:
Thanks for explaining this. This looks good as it looks now 👍